### ISSUE TITLE
SALTS error on install

### ISSUE THREAD

**Issue by ticsee** [tknorris/tknorris-beta-repo#44]
As soon as I install or start, it gives a SALTS error - "stream all the sources error check the log for more information". I am using 2.0.90 and here is the log https://logs.tvaddons.ag/phcswemeg . 
Could you please check what is happening? 
Thanks

**Comment by tknorris** [comment]
You are trying to run SALTS w/o my beta/release repo installed. That is not supported.

**Comment by ticsee** [comment]
Hm, I just checked as far as I understand and see it is installed...

**Comment by tknorris** [comment]
```
16:06:42 T:3024406464  NOTICE:  ADDONS: Using repository repository.exodus
16:06:42 T:3024406464  NOTICE:  ADDONS: Using repository repository.podgod
16:06:42 T:3024406464  NOTICE:  ADDONS: Using repository repository.xbmc.org
16:06:42 T:3024406464  NOTICE:  ADDONS: Using repository superrepo.kodi.jarvis.english
16:06:42 T:3024406464  NOTICE:  ADDONS: Using repository repository.xbmchub
16:06:42 T:3024406464  NOTICE:  ADDONS: Using repository repository.BlazeRepo
```

My repo is not installed according to your log.

**Comment by ticsee** [comment]
I have your release repository and vith that version is the same - that is relly odd
I see your point but I see it installed. I've tried couple of yimes to reinstall.....

**Comment by ticsee** [comment]
I will try to remove and install again everything and get back with new log. thank you

**Comment by tknorris** [comment]
Even if you have the release repo installed, 2.0.90 is not the current release version. You have done something really strange because the state you have gotten into shouldn't even be possible. Either install by beta repo and install SALTS via "Install from Repo" or install my release repo and isntall from Repo". Whatever you did, it wasn't that. ;-)

**Comment by ticsee** [comment]
I don't know what I have done, I think maybe I installed one version and didn't uninstalled addon before uninstalling repo, than installed second version. 
I see installed repo but its not there in the log. When I try to update gives me to choose from 2 or 3 files but gives same error. I've tried on another comp clean new installation and everything is fine. 
After some time playing - trying to remove it, didn't manage, so I removed KODI and all its files left behind, done fresh installation first beta SALTS and everything is working fine. 

Thank you very much for your help. :-)
